packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -242,6 +245,16 @@ export class ElectronMainApplication {
         return this.didUseNativeWindowFrameOnStart.get(webContents.id) ? 'native' : 'custom';
     }
 
+    protected showInitialWindow(): void {
+        if (!process.env.SKIP_INITIAL_WINDOW) {
```

Comment:
How can one configure this when there is a final bundled app?

Reply:
Please add the property to the electron frontend app config to make this configurable downstream.

https://github.com/eclipse-theia/theia/blob/f038673dc228485b34e8762686525d8e5f258afb/dev-packages/application-package/src/application-props.ts#L37-L45